logging/access.go

```diff
@@ -144,6 +173,10 @@ func LogAccess(entry *AccessEntry, additional map[string]interface{}) {
 		uri = entry.Request.RequestURI
 		if stripQuery {
 			uri = stripQueryString(uri)
+		} else if keys, ok := additional[KeyMaskedQueryParams].(map[string]bool); ok {
+			if len(keys) > 0 {
```

Comment:
see https://github.com/zalando/skipper/pull/3471/files#r2035043394

Reply:
the link is broken for me

Reply:
was this one https://github.com/zalando/skipper/pull/3471#discussion_r2035043394